app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ]  Also, this part, let's take an example from this:

```ruby
belongs_to :author, class_name: 'User'
belongs_to :post
```

Reply:
modified as required